pkg/nftables/util.go

```diff
@@ -55,10 +55,13 @@ func checksum(file string) (string, error) {
 	return hex.EncodeToString(h.Sum(nil)), nil
 }
 
-func assembleDestinationPortRule(common []string, protocol string, ports []string, comment string) string {
+func assembleDestinationPortRule(common []string, protocol string, ports []string, acceptLog bool, comment string) string {
 	parts := common
 	parts = append(parts, fmt.Sprintf("%s dport { %s }", protocol, strings.Join(ports, ", ")))
 	parts = append(parts, "counter")
+	if acceptLog {
+		parts = append(parts, "log prefix \"nftables-firewall-accepted: \"")
```

Comment:
I am a bit worried about ddos the firewall with to much log events of accepted flows. I think we must add a log limit rate here. Something like:

```
limit rate 3/minute
```
before the log statement

But according to the documentation i am unsure if this applies to log entries or packets as well

https://wiki.nftables.org/wiki-nftables/index.php/Limits

Reply:
I tried it: The rate limit applies to the whole line. 🙁

I changed the rules generation logic so that two lines are created; one rate limited for logging and one for counting and accepting.